spec/uploadcare/rails/api/rest/conversion_api_spec.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'uploadcare/rails/api/rest/conversion_api'
+
+module Uploadcare
+  module Rails
```

Comment:
Do these specs duplicate specs in the gem?

Reply:
yes, it does partially. Also, it checks correctness for several rails versions and methods presence for API classes. So, to guarantee that everything works properly with rails I added similar specs.